app/assets/config/orchid/manifest.js

```diff
@@ -1,5 +1,6 @@
 // Declare Orchid's assets for auto and pre-compilation
 // which can't be declared with portable relative paths from the generated app
 //= link_tree ../../images/orchid
+//= link_tree ../../images/favicon
```

Comment:
In the future I wouldn't re-commit things that are fixed in another branch to other branches like this. It conflates what branch/PR is handling what and could be error-prone if there are differences between the commits across branches.

If the commits are fixing errors like this, I recommend fixing them but just leaving the fix uncommitted on the branch where other work is being done because they should only be committed for merging on the one other branch.

It wasn't applicable in this case, but if we had gotten the other branch / PR merged before this was re-committed, I would recommend rebasing this branch off of `dev` so the branches merged into `dev` since this branch was started off of it are included as if you had created the branch off the updated `dev` branch.